### PR TITLE
#12446: Add GetUtilityPageIdentifiers for Manage Custom Pages to be excluded …

### PR DESCRIPTION
**Pull request by osrecio**
Added Interface and class to get collection of pages identifiers to be excluded from sitemap

### Description
Get default pages from config to be excluded in Sitemap.

### Fixed Issues (if relevant)
1. magento/magento2#12446: Remove /home from the sitemap.xml

### Manual testing scenarios
1. Create Sitemap
2. the pages like  `{domain}/home` `{domain}/no-route` no longer exists for sitemap

### Contribution checklist
 - [X] Pull request has a meaningful description of its purpose
 - [X] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [X] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by osrecio** [comment]
@vkublytskyi I should add `@SuppressWarnings(PHPMD.LongVariable)`  below PhpDocBlock? Or I can omit it?
```
/**
 * Sitemap cms page collection model
 *
 * @api
 * @since 100.0.2
 * @SuppressWarnings(PHPMD.LongVariable)
 */
```

**Comment by vkublytskyi** [comment]
@osrecio you may ignore Codacy failure if Magento static tests are green. Codacy is a recommendation, not a requirement.

**Comment by osrecio** [comment]
Some update @vkublytskyi ? Can I help you with something in this PR ?

File: app/code/Magento/Cms/Model/GetUtilityPageIdentifiers.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+namespace Magento\Cms\Model;
+
+use Magento\Cms\Api\GetUtilityPageIdentifiersInterface;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Store\Model\ScopeInterface;
+
+/**
+ * Utility Cms Pages
+ *
+ * @api
```
Comment (vkublytskyi) [inline app/code/Magento/Cms/Model/GetUtilityPageIdentifiers.php]: No need to mark class as an `@api` only interface is and API. Class as implementation is module internal details.

**Comment by vkublytskyi** [comment]
@osrecio I think we should remove `@api` annotation from a class as its interface already declared as module service contract and we can merge this PR.

**Comment by osrecio** [comment]
@vkublytskyi Deleted `@api`annotation :)

**Review by vkublytskyi — APPROVED** [review]

**Pull request merged**